terraform/aks/application.tf

```diff
@@ -31,8 +31,88 @@ module "application_configuration" {
   }
 }
 
+# Run database migrations
+# https://guides.rubyonrails.org/active_record_migrations.html#preparing-the-database
+# https://github.com/ilyakatz/data-migrate
+# Terraform waits for this to complete before starting web_application and worker_application
+resource "kubernetes_job" "migrations" {
+  metadata {
+    name      = "${var.service_name}-${var.environment}-migrations"
```

Comment:
the name needs to include the pr-number for review apps, otherwise they will have the same name i.e. "get-into-teaching-app-review-migrations"